### DISCUSSION TITLE
E' possibile creare DDT via API?

### DISCUSSION THREAD

**Discussion by stefanosilvestrinimia** [fattureincloud/api#330] [Q&A] [answered]
buongiorno, è possibile creare DDT via API?

Vorremo sapere se esiste la possibilità di creare un DDT via API (senza mandarlo via mail) in modo da farlo trovare dentro la dashboard di fatture in cloud. 

grazie

**Comment by emajo** [comment, marked as answer]
Ciao @stefanosilvestrinimia, certamente, per creare un DDT è necessario utilizzare la chiamata [Create Issued Document](https://developers.fattureincloud.it/api-reference#post-/c/-company_id-/issued_documents) passando il campo type settato a `delivery_note`, abbiamo inoltre anche [questa guida](https://developers.fattureincloud.it/docs/guides/invoice-creation#0%EF%B8%8F⃣-step-zero-the-document-type) che spiega in maniera più dettagliata i campi dell'issued document.

Grazie,
Emanuele Saccomandi.

**Reply by stefanosilvestrinimia** [reply]
grazie!

**Reply by stefanosilvestrinimia** [reply]
un'ulteriore domanda @emajo: come posso associare un 'document_id' di una proforma creata ad un DDT che vado a creare?

**Reply by emajo** [reply]
Ciao, purtroppo non è possibile fare ciò che chiedi, il flusso corretto è quello di andare a creare una proforma andando a popolare contestualmente anche i campi per il DDT, ovvero tutti quei campi che iniziano con `dn_` (_dn_number, dn_date ..._)

**Reply by stefanosilvestrinimia** [reply]
ok posso fare come indichi. Grazie. Mi riesci a dare un esempio dell'SDK PHP oppure indicare una pagina della documentazione per questi campi dn_ ? grazie ancora

**Reply by emajo** [reply]
Ciao, sì, però ti chiedo gentilmente di creare un altra discussion così che possa risultare utile anche agli altri utenti.

Grazie,
Emanuele.
